Makefile

```diff
@@ -38,13 +38,15 @@ endif
 gresource: .gresource.xml icon2.png
 	glib-compile-resources .gresource.xml --generate-source --target=src/resources.c
 
+droidcam-cli: LDLIBS += $(JPEG) $(LIBAV) $(LIBS)
 droidcam-cli: src/droidcam-cli.c $(SRC)
-	$(GXX) $(CC) $^ $(JPEG) $(LIBAV) $(LIBS) -o droidcam-cli
+	$(CXX) $(CPPFLAGS) $(CXXFLAGS) $^ -o $@ $(LDLIBS) $(LDFLAGS)
```

Comment:
Can we swap LDFLAGS and LDLIBS, logically IMO it makes more sense to specify the flags/search directories, followed by the actual includes/libs

Reply:
Sure!